okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -95,22 +98,30 @@ public void close(ExecutorService executor) {
 
   /** Creates a new builder for the given server host and port. */
   public static OkHttpChannelBuilder forAddress(String host, int port) {
-    return new OkHttpChannelBuilder(host, port);
+    return new OkHttpChannelBuilder(new InetSocketAddress(host, port));
```

Comment:
Instead of using the SocketAddress code path, would it be good to do:
```java
return forTarget("dns:///" + GrpcUtil.authorityFromHostAndPort(host, port));
```